lib/locales/es.js

```diff
@@ -11,7 +11,64 @@
+        'Sorry, there was a problem downloading your snapshot!': '¡La descarga de la instantánea falló!', // components/modebar/buttons.js:69
+        'Taking snapshot - this may take a few seconds': 'Capturando una instantánea - podría tardar unos segundos', // components/modebar/buttons.js:57
+        'Toggle Spike Lines': 'Mostrar/Ocultar Guías',                // components/modebar/buttons.js:547
+        'Toggle show closest data on hover': 'Activar/Desactivar mostrar el dato más cercano al pasar por encima', // components/modebar/buttons.js:352
+        'Turntable rotation': 'Rotación plana',                       // components/modebar/buttons.js:288
+        'Zoom': 'Modo Ampliar/Reducir',                               // components/modebar/buttons.js:85
+        'Zoom in': 'Ampliar',                                         // components/modebar/buttons.js:121
+        'Zoom out': 'Reducir',                                        // components/modebar/buttons.js:130
+        'close:': 'cierre:',                                          // traces/ohlc/transform.js:139
+        'high:': 'alza:',                                             // traces/ohlc/transform.js:137
+        'incoming flow count:': 'flujo de entrada',                   // traces/sankey/plot.js:142
+        'kde:': 'edp:',                                               // traces/violin/calc.js:73
+        'lat:': 'lat:',                                               // traces/scattergeo/calc.js:48
+        'lon:': 'lon:',                                               // traces/scattergeo/calc.js:49
+        'low:': 'baja:',                                              // traces/ohlc/transform.js:138
+        'lower fence:': 'límite inferior:',                           // traces/box/calc.js:134
+        'max:': 'max:',                                               // traces/box/calc.js:132
+        'mean ± σ:': 'media ± σ:',                                    // traces/box/calc.js:133
+        'mean:': 'media:',                                            // traces/box/calc.js:133
+        'median:': 'mediana:',                                        // traces/box/calc.js:128
+        'min:': 'min:',                                               // traces/box/calc.js:129
+        'open:': 'apertura:',                                         // traces/ohlc/transform.js:136
+        'outgoing flow count:': 'flujo de salida',                    // traces/sankey/plot.js:143
+        'q1:': 'q1:',                                                 // traces/box/calc.js:130
+        'q3:': 'q3:',                                                 // traces/box/calc.js:131
+        'source:': 'fuente:',                                         // traces/sankey/plot.js:140
+        'target:': 'destino:',                                        // traces/sankey/plot.js:141
+        'trace': 'traza:',                                            // plots/plots.js:305
```

Comment:
this one actually does *not* get a trailing quote - it's used in default trace names "trace 0" etc.

Reply:
(I mean trailing colon)